Problem to get Flathunter running

Unfortunately I have a problem to get Flathunter up and running. When I run "pytest" I get the following error message:


======================================================================== ERRORS =========================================================================
_________________________________________________________ ERROR collecting test/test_config.py __________________________________________________________
ImportError while importing test module '/home/planetdyna/test/test_config.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python2.7/dist-packages/six.py:709: in exec_
    exec("""exec _code_ in _globs_, _locs_""")
test/test_config.py:5: in <module>
    from flathunter.config import Config
E   ImportError: No module named flathunter.config
_________________________________________________ ERROR collecting test/test_crawl_ebaykleinanzeigen.py _________________________________________________
ImportError while importing test module '/home/planetdyna/test/test_crawl_ebaykleinanzeigen.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python2.7/dist-packages/six.py:709: in exec_
    exec("""exec _code_ in _globs_, _locs_""")
test/test_crawl_ebaykleinanzeigen.py:2: in <module>
    from flathunter.crawl_ebaykleinanzeigen import CrawlEbayKleinanzeigen
E   ImportError: No module named flathunter.crawl_ebaykleinanzeigen
__________________________________________________ ERROR collecting test/test_crawl_immobilienscout.py __________________________________________________
ImportError while importing test module '/home/planetdyna/test/test_crawl_immobilienscout.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python2.7/dist-packages/six.py:709: in exec_
    exec("""exec _code_ in _globs_, _locs_""")
test/test_crawl_immobilienscout.py:3: in <module>
    from flathunter.crawl_immobilienscout import CrawlImmobilienscout
E   ImportError: No module named flathunter.crawl_immobilienscout
_____________________________________________________ ERROR collecting test/test_crawl_immowelt.py ______________________________________________________
ImportError while importing test module '/home/planetdyna/test/test_crawl_immowelt.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python2.7/dist-packages/six.py:709: in exec_
    exec("""exec _code_ in _globs_, _locs_""")
test/test_crawl_immowelt.py:3: in <module>
    from flathunter.crawl_immowelt import CrawlImmowelt
E   ImportError: No module named flathunter.crawl_immowelt
_____________________________________________________ ERROR collecting test/test_crawl_wggesucht.py _____________________________________________________
ImportError while importing test module '/home/planetdyna/test/test_crawl_wggesucht.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python2.7/dist-packages/six.py:709: in exec_
    exec("""exec _code_ in _globs_, _locs_""")
test/test_crawl_wggesucht.py:3: in <module>
    from flathunter.crawl_wggesucht import CrawlWgGesucht
E   ImportError: No module named flathunter.crawl_wggesucht
________________________________________________ ERROR collecting test/test_gmaps_duration_processor.py _________________________________________________
/usr/lib/python2.7/dist-packages/_pytest/python.py:450: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
/usr/lib/python2.7/dist-packages/py/_path/local.py:701: in pyimport
    __import__(modname)
E     File "/home/planetdyna/test/test_gmaps_duration_processor.py", line 28
E   SyntaxError: Non-ASCII character '\xd0' in file /home/planetdyna/test/test_gmaps_duration_processor.py on line 29, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
________________________________________________ ERROR collecting test/test_googlecloud_idmaintainer.py _________________________________________________
ImportError while importing test module '/home/planetdyna/test/test_googlecloud_idmaintainer.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python2.7/dist-packages/six.py:709: in exec_
    exec("""exec _code_ in _globs_, _locs_""")
test/test_googlecloud_idmaintainer.py:4: in <module>
    from mockfirestore import MockFirestore
E   ImportError: No module named mockfirestore
_________________________________________________________ ERROR collecting test/test_hunter.py __________________________________________________________
ImportError while importing test module '/home/planetdyna/test/test_hunter.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python2.7/dist-packages/six.py:709: in exec_
    exec("""exec _code_ in _globs_, _locs_""")
test/test_hunter.py:2: in <module>
    import yaml
E   ImportError: No module named yaml
______________________________________________________ ERROR collecting test/test_id_maintainer.py ______________________________________________________
ImportError while importing test module '/home/planetdyna/test/test_id_maintainer.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python2.7/dist-packages/six.py:709: in exec_
    exec("""exec _code_ in _globs_, _locs_""")
test/test_id_maintainer.py:5: in <module>
    from flathunter.idmaintainer import IdMaintainer
E   ImportError: No module named flathunter.idmaintainer
________________________________________________________ ERROR collecting test/test_processor.py ________________________________________________________
ImportError while importing test module '/home/planetdyna/test/test_processor.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python2.7/dist-packages/six.py:709: in exec_
    exec("""exec _code_ in _globs_, _locs_""")
test/test_processor.py:2: in <module>
    import yaml
E   ImportError: No module named yaml
_____________________________________________________ ERROR collecting test/test_sender_telegram.py _____________________________________________________
ImportError while importing test module '/home/planetdyna/test/test_sender_telegram.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python2.7/dist-packages/six.py:709: in exec_
    exec("""exec _code_ in _globs_, _locs_""")
test/test_sender_telegram.py:1: in <module>
    import requests_mock
E   ImportError: No module named requests_mock
_____________________________________________________ ERROR collecting test/test_statistics_view.py _____________________________________________________
ImportError while importing test module '/home/planetdyna/test/test_statistics_view.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python2.7/dist-packages/six.py:709: in exec_
    exec("""exec _code_ in _globs_, _locs_""")
test/test_statistics_view.py:3: in <module>
    import yaml
E   ImportError: No module named yaml
______________________________________________________ ERROR collecting test/test_web_interface.py ______________________________________________________
ImportError while importing test module '/home/planetdyna/test/test_web_interface.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python2.7/dist-packages/six.py:709: in exec_
    exec("""exec _code_ in _globs_, _locs_""")
test/test_web_interface.py:3: in <module>
    import yaml
E   ImportError: No module named yaml
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 13 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

--- a/flathunter/crawl_ebaykleinanzeigen.py
+++ b/flathunter/crawl_ebaykleinanzeigen.py
@@ -79,13 +79,13 @@ class CrawlEbayKleinanzeigen(Crawler):
             try:
                 self.__log__.debug(tags[1].text)
                 rooms = re.match(r'(\d+)', tags[1].text)[1]
-            except IndexError:
+            except (IndexError, TypeError):
                 self.__log__.debug("Keine Zimmeranzahl gegeben")
                 rooms = "Nicht gegeben"
             try:
                 self.__log__.debug(tags[0].text)
                 size = tags[0].text
-            except IndexError:
+            except (IndexError, TypeError):
                 size = "Nicht gegeben"
                 self.__log__.debug("Quadratmeter nicht angegeben")
             details = {

